src/content/reference/docker-compose.mdx

```diff
@@ -309,18 +309,15 @@ scale: 2
 Configure resource [requests and limits](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits).
 
 ```yaml
-deploy:
-  resources:
-    reservations:
-      cpu: 300m
-      memory: 500Mi
-    limits:
-      cpu: 500m
-      memory: 800Mi
+resources:
+  requests:
+    cpu: 300m
+    memory: 500Mi
```

Comment:
I don't think is clear where in the compose this goes. Could you add a simple compose instead of the fragment?